pyvista/plotting/plotting.py

```diff
@@ -4011,6 +4011,16 @@ def show(self, title=None, window_size=None, interactive=True,
 
         >>> pl.show(jupyter_backend='ipygany', return_viewer=True)  # doctest:+SKIP
 
+        Obtain the camera position after closing the plotter
+
+        >>> pl = pv.Plotter()
+        >>> _ = pl.add_mesh(pyvista.Sphere())
```

Comment:
Caught by the hacky docstring tester :tada: 
```suggestion
        >>> _ = pl.add_mesh(pv.Sphere())
```